pages/regions.js

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { nanoid } from 'nanoid';
+
+import Layout from '../src/components/layout';
+import { Hero, H6, Icon } from '../src/components/jambonz-ui';
+import { getData } from '../src/lib/data';
+
+function Regions({ data }) {
+  const router = useRouter();
+  const cta = router.query.redirect;
+
+  return (
+    <section className="regions">
+      {data.regions.map((region) => {
+        return (
+          <div key={nanoid()} region={region.name} className="wrap regions__wrap">
+            <div className="regions__icon">
+              <Link href={cta === 'login' ? region.url : region.altUrl}>
+                <a>
+                  <Icon name={region.icon} subStyle={region.color} mainStyle="fill" />
+                </a>
+              </Link>
+            </div>
+            <div className="regions__title">
+              <H6>
+                {region.title}
+              </H6>
+            </div>
+            <div className="regions__text">
+              <Link href={cta === 'login' ? region.url : region.altUrl}>
+                <a>
+                  <H6>
+                    {region.text}
+                    <Icon name="ExternalLink" />
+                  </H6>
+                </a>
+              </Link>
+            </div>
+          </div>
+        );
+      })}
+    </section >
+  );
+}
+export default function RegionsPage({ data }) {
+  return (
+    <Layout siteData={data.site}>
```

Comment:
This layout needs to use the new `pink` altStyle for the `Hero` component. You can do that by adjusting the component structure like this:

```js
<Layout siteData={data.site}>
  <Hero data={data.regions.hero} altStyle="pink">
    <Regions data={data.regions} />
  </Hero>
</Layout>
```

We will also need this then added to the `&--pink` altStyle selector in the `_layout.scss` file to make the headline the jambonz pink color:

```sass
&--pink {
    // ...rest of code...
    h1 {
      color: $jambonz;
    }
    // ...rest of code...
```

And I also had to add a `margin-top` to the `regions` selector to readjust the spacing with this layout change:

```sass
.regions {
  text-align: center;
  // this is new
  margin-top: 64px;
```